tests/functional/testplan/testing/multitest/test_suite_decorators.py

```diff
@@ -26,22 +27,24 @@
 
 def pre1(name, self, env, result):
     result.equal(2, 2)
-    result.contain("testcase", name)
+    result.contain("case", name)
```

Comment:
Was this assertion originally failing?

Reply:
It does not harm because this testcase is not used to check report status, so no matter if the case/assertions is passed or failed.